### PR TITLE
Update FX rates

### PR DESCRIPTION
**Pull request by marcelmorgan**

### CODE REVIEW

**Comment by kenrick** [comment]
😃

**Comment by mcmorgan** [comment]
@kenrick it's about time 🤣

File: lib/forex/traders/jm/jmmb.rb [hunk 1]
```diff
@@ -2,29 +2,23 @@
   t.base_currency   = "JMD"
   t.name            = "Jamaica Money Market Brokers"
   t.twitter_handle  = "@JMMBGROUP"
-  t.endpoint        = "http://www.jmmb.com/full_rates.php"
+  t.endpoint        = 'https://jm.jmmb.com/rates-fees'
```
Comment (kenrick) [inline lib/forex/traders/jm/jmmb.rb]: You a single quote man now? 😆

**Comment by kenrick** [comment]
Looks good overall, but build is failing.

File: lib/forex/traders/jm/jmmb.rb (hunk 1 above)
Comment (mcmorgan) [reply lib/forex/traders/jm/jmmb.rb]: Lol, yes, for interpolated string

**Pull request merged**